src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -1007,6 +1007,42 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/snapshots/{id}/roles:
```

Comment:
We also probably want this for billing profiles

Reply:
As billing profiles are going away, I'm not sure we want to do the work to implement that. Sure, it wouldn't be hard, but we're not even going to support it in our UI, so why put in the effort?